src/main/scala/com/typesafe/conductr/sandbox/sbt/ConductRSandbox.scala

```diff
@@ -181,8 +200,14 @@ object ConductRSandbox extends AutoPlugin {
 
     val debugPorts = state.value.get(WithDebugAttrKey).fold(Set.empty[Int])(_ => debugPort.?.all(filter).value.flatten.toSet)
 
+    val roles = (conductrRoles in Global).value match {
+      case Nil           => Seq(BundleKeys.roles.?.map(_.getOrElse(Set.empty)).all(filter).value.reduce(_ ++ _))
```

Comment:
`flatten` will not work here because `flatten` returns an Iterator. In case of endpoints we expect a `Map` and in case of roles we expect a `Set`, but we would actually would get a `Seq`.